annotations/build.gradle.kts

```diff
@@ -12,6 +12,8 @@ java { sourceCompatibility = JavaVersion.VERSION_21 }
 
 repositories { mavenCentral() }
 
+extra["testcontainers.version"] = "1.19.7"
```

Comment:
The Testcontainers version is actually managed by Spring Boot, see [Spring Boot Reference Documentation](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#appendix.dependency-versions).

You can override the version, see [Spring Boot Gradle Plugin Reference Guide](https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/#managing-dependencies.dependency-management-plugin.customizing).